include/inviwo/core/io/rawvolumereader.h

```diff
@@ -54,25 +54,27 @@ class IVW_CORE_API RawVolumeReader : public DataReaderType<Volume> {
     virtual RawVolumeReader* clone() const override;
     virtual ~RawVolumeReader() = default;
 
-    virtual void setParameters(const DataFormatBase* format, ivec3 dimensions, bool littleEndian,
-                               DataMapper dataMapper, size_t byteOffset = 0u);
+    virtual void setParameters(const DataFormatBase* format, ivec3 dimensions,
```

Comment:
Generally all these parameters should be passed to the constructor.  I can't find any callers of this function let alone anyone overriding it...

Reply:
Will remove this function. It was already part of the initial v0.9.0 release.